acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuNodeConfigBuilder.java

```diff
@@ -317,6 +317,18 @@ public TekuNodeConfigBuilder withWritableKeystorePath(
     return this;
   }
 
+  public TekuNodeConfigBuilder withDefaultLogging() {
+    configMap.remove("log-destination");
+    return this;
+  }
+
+  public TekuNodeConfigBuilder withBadKeystorePath() {
+    LOG.debug("Xinterop-enabled=false");
```

Comment:
i'd be at least adding here that we're using a bad path as part of this debug..thats a pretty important breakage if you call this fn.